[2024-07-18 06:36:57,292] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: xcom_example_dag.pull_task 2024-06-25T00:00:00+00:00 [queued]>
[2024-07-18 06:36:57,339] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: xcom_example_dag.pull_task 2024-06-25T00:00:00+00:00 [queued]>
[2024-07-18 06:36:57,340] {taskinstance.py:1042} INFO - 
--------------------------------------------------------------------------------
[2024-07-18 06:36:57,341] {taskinstance.py:1043} INFO - Starting attempt 1 of 2
[2024-07-18 06:36:57,341] {taskinstance.py:1044} INFO - 
--------------------------------------------------------------------------------
[2024-07-18 06:36:57,413] {taskinstance.py:1063} INFO - Executing <Task(PythonOperator): pull_task> on 2024-06-25T00:00:00+00:00
[2024-07-18 06:36:57,439] {standard_task_runner.py:52} INFO - Started process 512 to run task
[2024-07-18 06:36:57,457] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'xcom_example_dag', 'pull_task', '2024-06-25T00:00:00+00:00', '--job-id', '714', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/xcom.py', '--cfg-path', '/tmp/tmpctylazck', '--error-file', '/tmp/tmpmifor0b1']
[2024-07-18 06:36:57,475] {standard_task_runner.py:77} INFO - Job 714: Subtask pull_task
[2024-07-18 06:36:57,880] {logging_mixin.py:104} INFO - Running <TaskInstance: xcom_example_dag.pull_task 2024-06-25T00:00:00+00:00 [running]> on host a868c7bacc12
[2024-07-18 06:36:58,145] {taskinstance.py:1257} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=xcom_example_dag
AIRFLOW_CTX_TASK_ID=pull_task
AIRFLOW_CTX_EXECUTION_DATE=2024-06-25T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2024-06-25T00:00:00+00:00
[2024-07-18 06:36:58,186] {logging_mixin.py:104} INFO - Pulled message: Hello from push_function!
[2024-07-18 06:36:58,188] {python.py:118} INFO - Done. Returned value was: None
[2024-07-18 06:36:58,253] {taskinstance.py:1166} INFO - Marking task as SUCCESS. dag_id=xcom_example_dag, task_id=pull_task, execution_date=20240625T000000, start_date=20240718T063657, end_date=20240718T063658
[2024-07-18 06:36:58,460] {taskinstance.py:1220} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-07-18 06:36:58,494] {local_task_job.py:146} INFO - Task exited with return code 0
